voluspa.py

```diff
@@ -36,8 +36,13 @@
 from aiocache import caches
 caches.set_config(CONFIG.Voluspa.cache)
 
+# New as of discordpy 1.5.1
+intents = discord.Intents.default()
+intents.members = True
+intents.presences = True
+
 # Setup Initial Stuff
-client = discord.Client()
+client = discord.Client(intents=intents)
```

Comment:
I read the discord.py docs on this. Seems like a bit of a yikes in terms of rate limiting etc, but fine for us. Thanks for handling since this would've caused downtime that I can't remember lol

Reply:
Of course, no prob at all.